fastify.d.ts

```diff
@@ -312,6 +312,23 @@ declare namespace fastify {
     trailers: object
   }
 
+  /**
+   * Server listen options
+   */
+  interface ListenOptions {
+    port?: number;
+    host?: string;
+    backlog?: number;
+    path?: string;
```

Comment:
Sorry, I don't understand this `path` property: it is not tested in the typescript checks.
Maybe it refers to `sockFile`? Could we change this input name to be more clear?

Reply:
@Eomm 
`path` if from [options](`https://nodejs.org/api/net.html#net_server_listen_options_callback`) for Node.js `server.listen`, so I think it may be better to keep consistency.

Reply:
I understand the choice now. I'm conflicted only because in others ts signatures that parameter has been called `sockFile`. I know that this renaming impact on the `Object.assign(options, firstArg)` but I think we should be clear for users.

So, it could be enough write it on the example? Something like "the options object will be passed to the Node.js server listen".

Reply:
Sure, will update readme

Reply:
Added in https://github.com/fastify/fastify/pull/1463/files#diff-a5b695b68b867140473b0778bcfe36edR388